subprojects/core-api/src/main/java/org/gradle/api/internal/lambdas/SerializableLambdas.java

```diff
@@ -56,6 +57,10 @@ public static <T, U, R> BiFunction<T, U, R> bifunction(SerializableBiFunction<T,
         return f;
     }
 
+    public static <T> Supplier<T> spec(SerializableSupplier<T> supplier) {
```

Comment:
:x: Should this method be called `spec`? I'd prefer `supplier`.

Reply:
😮

Reply:
Thanks, Mike, crisis adverted! 😄